### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by nnpcYvIVl**

### CODE REVIEW

File: src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs [hunk 1]
```diff
@@ -11038,7 +11038,7 @@ long M(int i)
             Assert.DoesNotContain($"error CS0078", output, StringComparison.Ordinal);
             Assert.DoesNotContain($"warning CS0078", output, StringComparison.Ordinal);
 
-            // Diagnostic '{0}: {1}' was programmatically suppressed by a DiagnosticSuppressor with suppresion ID '{2}' and justification '{3}'
+            // Diagnostic '{0}: {1}' was programmatically suppressed by a DiagnosticSuppressor with suppression ID '{2}' and justification '{3}'
```
Comment (cston) [inline src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs]: Please update `SuppressionDiagnosticDescriptorMessage` in CodeAnalysisResources.resx as well.

Comment (nnpcYvIVl) [reply src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs]: @cston Looks like `SuppressionDiagnosticDescriptorMessage` in CodeAnalysisResources.resx is already correct.

Comment (sharwell) [reply src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs]: Looks like whenever the resx file was updated, the designer for it wasn't updated to match. AFAICT it's all consistent now with this PR. 👍

**Comment by cston** [comment]
@dotnet/roslyn-compiler for review.

**Comment by cston** [comment]
Thanks for the contribution!

**Pull request merged**